library/src/androidTest/kotlin/com/chuckerteam/chucker/SpanUtilTest.kt

```diff
@@ -0,0 +1,92 @@
+package com.chuckerteam.chucker
+
+import android.annotation.SuppressLint
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.chuckerteam.chucker.internal.support.SpanTextUtil
+import com.google.common.truth.Truth
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+public class SpanUtilTest {
+    @SuppressLint("CheckResult")
+    @Test
+    public fun json_can_have_null_value() {
+
```

Comment:
Hi @ArjanSM 

Hope you are doing well. I want to inform you that I changed the method's name in my test class and again I am using underscore instead of space as this convention is not supported in android runtime tests.
You can check [this link](https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods) for more information.